src/components/Match/MatchStory.jsx

```diff
@@ -174,6 +174,8 @@ const evaluateSentiment = (event, lastMessage) => {
 
   if (message.split(' ').length > 10) {
     sentiment.push('long');
+  } else if (['XD', ':D', 'LOL'].indexOf(message.replace('?', '').toUpperCase()) !== -1) {
```

Comment:
es6 has `.includes()`

Reply:
Doesn't work on Internet Explorer, will the toolchain handle it properly?

Reply:
yes, babel should polyfill it for older browsers